package.json

```diff
@@ -28,7 +28,7 @@
     "repl.js"
   ],
   "scripts": {
-    "postinstall": "node --eval 'if (require(\"./package.json\").name === \"coffee-script\") { var red, yellow, cyan, reset; red = yellow = cyan = reset = \"\"; if (!process.env.NODE_DISABLE_COLORS) { red = \"\\x1b[31m\"; yellow = \"\\x1b[33m\"; cyan = \"\\x1b[36m\"; reset = \"\\x1b[0m\"; } console.warn(red + \"CoffeeScript has moved!\" + reset + \" Please update references to \" + yellow + \"\\\"coffee-script\\\"\" + reset + \" to use \" + yellow + \"\\\"coffeescript\\\"\" + reset + \" (no hyphen) instead.\"); console.warn(\"Also, a new major version has been released under the \" + yellow + \"coffeescript\" + reset + \" name on NPM. This new release targets modern JavaScript, with minimal breaking changes. Learn more at \" + cyan + \"http://coffeescript.org\" + reset + \".\"); console.warn(\"\"); }'",
+    "postinstall": "node --eval \"if (require('./package.json').name === 'coffee-script') { var red, yellow, cyan, reset; red = yellow = cyan = reset = ''; if (!process.env.NODE_DISABLE_COLORS) { red = '\\x1b[31m'; yellow = '\\x1b[33m'; cyan = '\\x1b[36m'; reset = '\\x1b[0m'; } console.warn(red + 'CoffeeScript has moved!' + reset + ' Please update references to ' + yellow + '\\\"coffee-script\\\"' + reset + ' to use ' + yellow + '\\\"coffeescript\\\"' + reset + ' (no hyphen) instead.'); console.warn('Also, a new major version has been released under the ' + yellow + 'coffeescript' + reset + ' name on NPM. This new release targets modern JavaScript, with minimal breaking changes. Learn more at ' + cyan + 'http://coffeescript.org' + reset + '.'); console.warn(''); }\"",
```

Comment:
have you tried `\\\"coffee-script\\\"` and confirm it works on both platforms?

Reply:
@hajjem-ayoub Does it works on your local with this fix?

Reply:
yes tried it only with windows VM, i can also check on mac

Reply:
This shouldn't work.

Reply:
i also tried to npm link it in mac and it seems to work as well, it shows the message correctly at least

Reply:
@rossanmol just to debug it, can you tell me how did you test it?

Reply:
run `node --eval` in cmd